commands/util/binary.js

```diff
@@ -0,0 +1,70 @@
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("binary")
+    .setDescription("A command with binary conversion and addition functionality")
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("convert")
+        .setDescription("Convert a number to binary")
+        .addIntegerOption(option =>
+          option
+            .setName("number")
+            .setDescription("Pick a number to convert into binary")
+            .setRequired(true)
+        )
+    )
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("add")
+        .setDescription("Add two numbers")
+        .addIntegerOption(option =>
+          option
+            .setName("first")
+            .setDescription("The first number")
+            .setRequired(true)
+        )
+        .addIntegerOption(option =>
+          option
+            .setName("second")
+            .setDescription("The second number")
+            .setRequired(true)
+        )
+    ),
+  async execute(interaction) {
+    const subcommand = interaction.options.getSubcommand();
+
+    if (subcommand === "convert") {
+      const integer = interaction.options.getInteger("integer");
+      const binary = integer.toString(2);
+
+      const embed = new EmbedBuilder()
+        .setTitle("Binary Converter")
+        .addFields(
+            {name: 'Number', value: `**${integer}**`, inline: true},
+            {name: 'Binary', value: `**${binary}**`, inline: true},
+        )
+        .setColor("BLUE");
```

Comment:
Remove color for now. Let's stay consistent with our color scheme until we find a color that we can use for all embeds